var/spack/repos/builtin/packages/py-tensorflow-estimator/package.py

```diff
@@ -26,6 +29,11 @@ class PyTensorflowEstimator(Package):
 
     extends('python')
 
+    depends_on('py-keras@2.7.0:2.7', type=('build', 'run'), when='@2.7.0')
+    depends_on('py-keras@2.6.0:2.6', type=('build', 'run'), when='@2.6.0')
```

Comment:
I don't see any deps declared other than TF

Reply:
As I said in https://github.com/spack/spack/pull/28093#issuecomment-1002196071:

> I determined the py-keras dependencies from building.